src/components/Navigation/Navigation.jsx

```diff
@@ -37,32 +32,26 @@ export default function Navigation(){
 		}
 	];
 
-
-	const links       = linkData.map(RENDER.item);
-	const ariaOpen    = open.toString();
+	const links      	  = linkData.map(RENDER.item);
+	const ariaExpanded    = expanded.toString();
 
 	return (
-		<nav>
-			<button 
-				role="switch"
-				aria-checked={ ariaOpen }
-				aria-controls="navigation__links"
-				onClick={ toggleOpen }
-			>
-				<span>
-					Menu
-				</span>
-				<span>
-					Close
-				</span>
-			</button>
+		<nav className={ s.wrapper }>
+			<NavProvider>
+				<NavToggle 
+					setExpanded={ setExpanded }
+				/>
+			</NavProvider>
 			<ul 
 				id="navigation__links"
-				aria-expanded={ ariaOpen }
+				className={ s.itemContainer }
+				aria-expanded={ ariaExpanded }
 			>
 				{ links }
 			</ul>
-			<Socials />
+			<div className={ s.socials }>
```

Comment:
Can we lose this wrapper? I don't think we need it.

Reply:
Oop, it's still here 😬 - if there's a reason that `<Socials />` needs to be wrapped then let's totally discuss that when we go over stuff later.